server/graphql/common/expenses.ts

```diff
@@ -1819,13 +1819,13 @@ export async function submitExpenseDraft(
     throw new Unauthorized('Expense can not be edited.');
   }
 
-  const userIsOriginalPayee = originalPayee && req.remoteUser?.isAdminOfCollective(originalPayee);
-  const userIsAuthor = req.remoteUser?.id === existingExpense.UserId;
+  const userIsOriginalPayee: boolean = Boolean(originalPayee) && req.remoteUser?.isAdminOfCollective(originalPayee);
+  const userIsAuthor = Boolean(req.remoteUser) && req.remoteUser?.id === existingExpense.UserId;
```

Comment:
No need to double check

```suggestion
  const userIsAuthor = Boolean(req.remoteUser) && req.remoteUser.id === existingExpense.UserId;
```